content/api/plugins/writing-a-plugin.md

```diff
@@ -141,8 +141,9 @@ Cypress.
 
 <strong class="alert-header">Node version</strong>
 
-Keep in mind - code executed in plugins **may** be executed by the Node version
-that comes bundled in Cypress itself.
+As of version 9.0.0, Cypress will use the same version of node that was used to launch the cypress App via the CLI.
+
+Prior to version 9.0.0, Cypress would use the version of node bundled with Cypress itself.
```

Comment:
We don't keep notes like this in our docs for previous versions, the docs always reflect current version. In some cases we have a 'History' section in the file to explain changes

Can you update this section to reflect the truth of the new version?